### PR TITLE
Create user branch

### PR DESCRIPTION
**Pull request by simionescuraluca**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -56,6 +69,8 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
```
Comment (dragosslujeru) [inline pom.xml]: review these extra lines

File: pom.xml [hunk 2]
```diff
@@ -16,9 +16,22 @@
 
 	<properties>
 		<java.version>1.8</java.version>
+
```
Comment (dragosslujeru) [inline pom.xml]: also this

File: pom.xml (hunk 1 above)
Comment (simionescuraluca) [reply pom.xml]: fixed

File: pom.xml (hunk 2 above)
Comment (simionescuraluca) [reply pom.xml]: fixed

**Review by dragosslujeru — APPROVED** [review]

**Pull request merged**